sympy/physics/mechanics/tests/test_actuator.py

```diff
@@ -849,3 +855,111 @@ def test_to_loads(self):
+        ===================
+        The equations of motion for the block are derived considering
+        the applied force F, the mass m, the gravitational constant g, and the friction forces
+        as described above.
+
+        The general form of the equation of motion is:
+
+            F - F_friction = m * xdd
+
+        where F_friction is given by the friction force equation f_f(v).
+
+        Coordinate System
+        =================
+
+                x
+            --->
+            |
+            v y
+
+                    ---> v+ or v-
+                  _______
+              F   |     |       |
+            --->  |  m  |       | g
+                  |_____|       v
+                         <--- F_k = f_c * sign(v) + (f_max - f_c) * exp(-(v / v_s)^2) + sigma * v
+             N = m*g ^
+                     |
+
+        Velocity Cases
+        ==============
+
+        For positive velocity (v = 3.141):
+
+            F - m * Derivative(3.141, t) + (g * m * mu_k * sign(sqrt(q1(t)**2) / q1(t))
+            + 3.141 * sigma * sqrt(q1(t)**2) / q1(t)
+            + (-g * m * mu_k + g * m * mu_s) * exp(-9.865881 / v_s**2)) * q1(t) / sqrt(q1(t)**2)
+
+        For negative velocity (v = -3.141):
+
+            F - m * Derivative(-3.141, t) + (-g * m * mu_k * sign(sqrt(q1(t)**2) / q1(t))
+            - 3.141 * sigma * sqrt(q1(t)**2) / q1(t)
+            + (-g * m * mu_k + g * m * mu_s) * exp(-9.865881 / v_s**2)) * q1(t) / sqrt(q1(t)**2)
+
+        The test compares the derived equations of motion with the expected ones to
+        ensure the friction model is implemented correctly.
+
+        """
+        # Define Dynamics symbols and parameters
+        q1, u1 = dynamicsymbols('q1 u1')
+
+        # Mass, gravity constant, friction coefficient, coefficient of sliding friction, viscous_coefficient
+        m, g, mu_k, mu_s, v_s, sigma, F = symbols('m g mu_k mu_s v_s sigma F')
+
+        # Define the reference frame
+        N = ReferenceFrame('N')
+        O = Point('O')
+        P = O.locatenew('P', q1 * N.x)
+        O.set_vel(N, 0)
+        P.set_vel(N, u1 * N.x)
+
+        particle = Particle('particle', P, m) # Block
+
+        friction = CoulombKineticFriction(
+            mu_k=mu_k,
+            mu_s=mu_s,
+            normal_force=m * g,
+            pathway=LinearPathway(O, P),
+            v_s=v_s,
+            viscous_coefficient=sigma,
+        )
+
+        system = System(N, O)
+        system.q_ind = [q1]
+        system.u_ind = [u1]
+        system.kdes = [q1.diff() - u1]
+        system.add_bodies(particle)
+        system.add_actuators(friction)
+        system.apply_uniform_gravity(-g * N.y)
+        system.add_loads(Force(particle, F * N.x))
+
+        eoms = system.form_eoms()
+
+        # Positive velocity case
+        expected_positive = F - m * 0 + (g * m * mu_k * sign(sqrt(q1**2) * 3.141 / q1) + sigma * sqrt(q1**2) * 3.141 / q1 + (-g * m * mu_k + g * m * mu_s) * exp(-9.865881 / v_s**2)) * q1 / sqrt(q1**2)
```

Comment:
I wouldn't expect a `sign` in your expected result. Maybe you'll have to create your symbols with the assumption `real=True`.